examples/tutorial_examples/plot_001_afq_api.py

```diff
@@ -55,14 +55,17 @@
 # ---------------------------------------
 # We make create a `tracking_params` variable, which we will pass to the
 # GroupAFQ object which specifies that we want 25,000 seeds randomly
-# distributed in the white matter.
+# distributed in the white matter. We also set num_chunks to True,
+# which will use ray to parallelize the tracking across each core.
+# This can be removed to process in serial.
```

Comment:
Also could add here: 
```suggestion
# This can be removed to process in serial or set to use a particular distribution of work by setting `n_chunks` to an integer number.
```